sgbackend/mail.py

```diff
@@ -66,7 +71,12 @@ def send_messages(self, emails):
 
     def _build_sg_mail(self, email):
         mail = Mail()
-        mail.set_from(Email(email.from_email))
+        from_name, from_email = rfc822.parseaddr(email.from_email)
+        # Python sendgrid client should improve
+        # sendgrid/helpers/mail/mail.py:164
```

Comment:
Nice! We will track here: https://github.com/sendgrid/sendgrid-python/issues/277